test/conformance/testdata/conformance.txt

```diff
@@ -192,6 +192,7 @@ test/e2e/kubectl/kubectl.go: "should update a single-container pod's image"
 test/e2e/kubectl/kubectl.go: "should create a job from an image, then delete the job"
 test/e2e/kubectl/kubectl.go: "should support proxy with --port 0"
 test/e2e/kubectl/kubectl.go: "should support --unix-socket=/path"
+test/e2e/lifecycle/resize_nodes.go: "should be able to delete nodes"
```

Comment:
This should have not been allowed in the lest and should have failed our conformance list verification test